kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/_Role.kt

```diff
@@ -0,0 +1,14 @@
+package com.kotlindiscord.kord.extensions.utils
+
+import dev.kord.core.entity.Role
+
+/**
+ * Checks whether a [Role] can interact with another [Role] by comparing their [rawPosition]s.
+ *
+ * Throws an [IllegalArgumentException] when the roles are not from the same guild.
+ */
+public fun Role.canInteract(role: Role): Boolean {
+    if (role.guildId != guildId)
+        throw IllegalArgumentException("canInteract can only be called within the same guild!")
+    return role.rawPosition < rawPosition
+}
```

Comment:
Newlines. Both in the function, and at the end of the file. Also, this conditional is on more than one line, so it needs braces.